### DISCUSSION TITLE
DBB 3.0 Upgrade

### DISCUSSION THREAD

**Discussion by mhardin18** [IBM/dbb-zappbuild#549] [Q&A] [answered]
@dennis-behm 
@M-DLB 

With the new DBB 3.0 available, should we anticipate needing changes in zAppBuild if we have no intention of implementing the zBuilder integrated build framework?

**Comment by dennis-behm** [comment, marked as answer]
Hi Matthew,  DBB 3.0 is fully compatible with the zAppBuild build scripts. No changes needed.
